nano/node/vote_processor.hpp

```diff
@@ -75,10 +75,7 @@ class vote_processor final
 	void run_batch (nano::unique_lock<nano::mutex> &);
 
 private:
-	std::size_t const max_votes;
-
-	using entry_t = std::pair<std::shared_ptr<nano::vote>, std::shared_ptr<nano::transport::channel>>;
-	nano::fair_queue<entry_t, nano::rep_tier> queue;
+	nano::fair_queue<std::shared_ptr<nano::vote>, nano::rep_tier> queue;
```

Comment:
Was this found in performance profiling or could there be a test added to ensure correct behavior?

Reply:
This was found by network simulation tests (https://ct.bnano.info). I'm not sure if those long running tests are a good fit for core tests suite, but for this very reason I recently made a bot that automatically shows results under PRs https://github.com/pwojcikdev/nano-continuous-testing-bot